app/commands/tooling_job/cancel.rb

```diff
@@ -0,0 +1,75 @@
+module ToolingJob
+  class Cancel
+    include Mandate
+
+    initialize_with :submission_uuid, :type
+
+    def call
+      return unless pending_dynamodb_job
+
+      update_dynamodb
+      update_submission
+    end
+
+    private
+    memoize
+    def client
+      ExercismConfig::SetupDynamoDBClient.()
+    end
```

Comment:
Really minor comment, but helps with consistency. I tend to put these methods that don't do much at the bottom of the file, and then try to keep the other methods loosely in the order that they appear in the code.

So I'd have them ordered as:
```
pending_dynamodb_job
update_dynamodb
update_submission
client
```